src/apps/stunclient/stunclient.c

```diff
@@ -62,37 +62,36 @@ static int run_stunclient(const char *rip, int rport, int *port, bool *rfc5780,
                           bool change_port, int padding) {
 
   ioa_addr remote_addr;
-  int new_udp_fd = -1;
 
   memset((void *)&remote_addr, 0, sizeof(ioa_addr));
   if (make_ioa_addr((const uint8_t *)rip, rport, &remote_addr) < 0) {
-    err(-1, NULL);
+    err(-1, nullptr);
```

Comment:
the `NULL` -> `nullptr` changes are only in C++ code, where `nullptr` has important properties that `NULL` lacks (e.g. different, non-integral type, helping to avoid passing `NULL` to an `int` parameter.)